### ISSUE TITLE
How to train from scratch?

### ISSUE THREAD

**Issue by Melchoirr** [shiyu-coder/Kronos#128]
Does loading config only to randomly initialize parameters and finetune equal to training from scratch?

**Comment by shiyu-coder** [comment]
Pretraining requires a very large amount of data, and a fine tuned dataset is not enough. In addition, pretraining requires a higher learning rate and longer training time.

**Comment by Melchoirr** [comment]
Thank you～
